Source/Initialization/WarpXInitData.cpp

```diff
@@ -266,6 +267,74 @@ WarpX::ComputePMLFactors ()
     }
 }
 
+void
+WarpX::ComputeMaxStep ()
```

Comment:
We call this new function only once in `InitData`, right?

Because I was wondering with that logic, if we would update `stop_time` and `max_step` multiple times due to rounding errors. Otherwise seems legit :)

Reply:
Yes it's currently only called in `WarpX::InitData ()`.